### PR TITLE
Feature/financial management

### PR DESCRIPTION
**Pull request by LordST95**

### CODE REVIEW

File: ketabyar/settings.py [hunk 1]
```diff
@@ -151,9 +152,9 @@
 
 # Sending email settings
 EMAIL_USE_TLS = True
-EMAIL_HOST = 'smtp.mailgun.org'
-EMAIL_HOST_USER = 'postmaster@sandboxf273f5445e524eeb84687a2451867baf.mailgun.org'
-EMAIL_HOST_PASSWORD = '5766f175d38a60781742ef19bac1567a'
+EMAIL_HOST = 'smtp.gmail.com'
+EMAIL_HOST_USER = 'ps4kadeh@gmail.com'
```
Comment (mh-izadi) [inline ketabyar/settings.py]: we should have a mail account with relevant name to ketabyar

File: accounts/models.py [hunk 2]
```diff
@@ -15,16 +15,7 @@ class Member(AbstractUser):
     age = models.IntegerField(null=True, blank=True)
     job = models.CharField(max_length=100, null=True)
     education = models.CharField(max_length=100, null=True)
-    # address = models.ForeignKey(to='Address', on_delete=models.CASCADE, null=True)
```
Comment (mh-izadi) [inline accounts/models.py]: why remove address?

File: ketabyar/settings.py (hunk 1 above)
Comment (mh-izadi) [reply ketabyar/settings.py]: you can use ketabyarApp@gmail instead. i have created that account in gmail.

Comment (LordST95) [reply ketabyar/settings.py]: that's so good, plz give me the password of that account and remember we should enable third weak party access on it :+1:

File: accounts/models.py (hunk 2 above)
Comment (LordST95) [reply accounts/models.py]: address variable was created in the first line of that function

File: ketabyar/settings.py (hunk 1 above)
Comment (mh-izadi) [reply ketabyar/settings.py]: password is same with former acount

**Comment by LordST95** [comment]
this feature still have so many things to do :))

File: FinancialManagement/views.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        context = super().get_context_data(**kwargs)
+        if self.request.user.is_anonymous:
+            raise PermissionDenied
+        try:
+            user = InvitationLink.objects.get(invitee=self.request.user)
+            context['invitationLink'] = InvitationLink.objects.get(invitee=user)
+            return context
+        except ObjectDoesNotExist:
+            invitation_link = InvitationLink.objects.create(invitee=self.request.user)
+            tekrari = True
```
Comment (nahmadian) [inline FinancialManagement/views.py]: Better variable name? like isInvitedBefore.

**Pull request merged**